src/MercadoPago/Client/Order/OrderRefundClient.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace MercadoPago.Client.Order
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using MercadoPago.Error;
+    using MercadoPago.Http;
+    using MercadoPago.Resource.Order;
+    using MercadoPago.Serialization;
+
+    /// <summary>
+    /// Client that use the Order APIs.
+    /// </summary>
+    public class OrderRefundClient : MercadoPagoClient<OrderRefund>
+    {
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OrderRefundClient"/> class.
+        /// </summary>
+        /// <param name="httpClient">The http client that will be used in HTTP requests.</param>
+        /// <param name="serializer">
+        /// The serializer that will be used to serialize the HTTP requests content
+        /// and to deserialize the HTTP response content.
+        /// </param>
+        public OrderRefundClient(IHttpClient httpClient, ISerializer serializer)
+            : base(httpClient, serializer)
+        {
+        }
+
```

Comment:
[req] Vejo que a função está vazia, outras abaixo também, elas deveriam instanciar o Refund Client, não?
```suggestion
        public OrderRefundClient(IHttpClient httpClient, ISerializer serializer)
            : base(httpClient, serializer)
        {
        }

```

Reply:
Não precisa, vide demais implementações de client.